src/providers/database.ts

```diff
@@ -87,7 +87,7 @@ export function _instanceWithOptions(
 }
 
 export class InstanceBuilder {
-  /* @internal */
+  /* @hidden */
```

Comment:
`/** @hidden */`

Reply:
Good catch, fixed in https://github.com/firebase/firebase-functions/pull/511/commits/e69011bd9b8e445720d4457d3cecf760b02a600e.